### ISSUE TITLE
not able to install 'accessibility' package.

### ISSUE THREAD

**Issue by Mehrnaz-accessibility** [ipeaGIT/accessibility#55]
Dear authors,

Thank you so much for publishing the useful and practical package of 'accessibility'. 
I'm willing to use this package for a project. I've already prepared the data. But I got stuck into the first step, installing the package. I have tried these 2 ways. But still not succeeded!
1.
install.packages("accessibility")
2.
install.packages("remotes")
remotes::install_github("ipeaGIT/accessibility")
here is the comment I received:
Error: Failed to install 'accessibility' from GitHub:
  ! System command 'Rcmd.exe' failed

It will be very appreciated if you help me with solving this issue.

**Comment by dhersz** [comment]
Hi. I wasn't able to reproduce the error locally, installation ran successfully here.

Is the error message the same when using `install.packages("accessibility")`?

This does not seem related to the package. Rather, it seems to be an issue with your R installation. Have you tried searching for a solution in StackOverflow, Google, etc?

**Comment by Mehrnaz-accessibility** [comment]
Hi Daniel, Thank you very much for your prompt response! 
Yes the issue repeats when using install.packages("accessibility")?

![image](https://github.com/ipeaGIT/accessibility/assets/172813956/3b1a5335-860d-4ecb-9439-c5deae91b97b)

**Comment by Mehrnaz-accessibility** [comment]
I could solve the issue! What worked for me was: 
uninstalling R, Rtools, and Rstuido. Then downloading and run as administrator in this order: 1)R  2)Rtools and 3)Rstuido!

**Comment by dhersz** [comment]
Glad to know :)
